src/internal/Esp32_i2s.c

```diff
@@ -58,6 +58,7 @@
 
 #include "Esp32_i2s.h"
 #include "esp32-hal.h"
+#include "rom/lldesc.h"
```

Comment:
minor, related header is usually the last thing included in the c/cpp file.

Reply:
OK, changed